Api/CreateContainer.ts

```diff
@@ -12,6 +12,12 @@ import UserService from './Src/Services/User';
 import UserController from './Src/Controllers/User';
 import userRoutes from './Src/Routes/User';
 
+import PasswordResetTokenModel from './Src/Models/PasswordResetToken';
+import PasswordService from './Src/Services/Password';
+import PasswordController from './Src/Controllers/Password';
+import PasswordRoutes from './Src/Routes/Password';
+import { Repository } from './Src/Repositories/Repository';
```

Comment:
We decided to name files like: Services/PasswordService and so on.

Reply:
changed